iot-hub/Samples/device/AndroidSample/things/build.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'com.android.application'
+
+//***********************************************************************************************//
+def IOTHUB_DEVICE_CONNECTION_STRING_ENV_VAR_NAME = project.hasProperty('DeviceConnectionString') ? '"'+project.property('DeviceConnectionString')+'"' : '"Define Device Connection"'
+def STRING='String'
+//***********************************************************************************************//
+
+android {
+    compileSdkVersion 27
+
+
+    defaultConfig {
+        applicationId "android.iot.sdk.azure.microsoft.com.things"
+        minSdkVersion 24
+        targetSdkVersion 27
+        versionCode 1
```

Comment:
is there a way to mark targetSdkVersion as min + any future? This is possible in POM and we should try to see if it possible in Android as well.

Reply:
not exactly sure why you want to use min+?  My understanding is that minSdkVersion defines the min, compileSdkVersion is the one you compiled with and targetSdkVersion is anything in between which is used by Android to provide forward compatibility without applying behavior changes...  Ideal values should be: minSdkVersion (lowest possible) <= 
    targetSdkVersion == compileSdkVersion (latest SDK)

I am updating both samples to min 24, target and compile 28.